front/src/common/BootstrapSNCF/InputGroupSNCF.tsx

```diff
@@ -76,6 +80,29 @@ export default function InputGroupSNCF({
     });
   }, [type, options]);
 
+  const handleOnChange = useCallback(
+    (event: ChangeEvent<HTMLInputElement>) => {
+      const eventValue = Number(event.target.value);
+      if (event.target.value !== '' && limitDecimal !== undefined) {
+        const limitedValue = stripDecimalDigits(eventValue, limitDecimal);
+        handleType({ type: selected.id, value: limitedValue });
+      } else {
+        handleType({ type: selected.id, value: eventValue });
+      }
+    },
+    [handleType, selected, limitDecimal, value]
+  );
+
+  const inputValue = useMemo(() => {
+    if (value !== undefined && !disabled) {
+      if (limitDecimal && value !== '') {
+        return stripDecimalDigits(Number(value), limitDecimal);
+      }
+      return value;
+    }
+    return undefined;
```

Comment:
Use "" otherwise you will have an error in your console
```suggestion
    return '';
```